spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/factory/HystrixGatewayFilterFactory.java

```diff
@@ -185,6 +192,16 @@ public void setFallbackUri(URI fallbackUri) {
 			this.fallbackUri = fallbackUri;
 		}
 
+		public Integer getTimeoutInMilliseconds() {
+			return timeoutInMilliseconds;
+		}
+
+		public Config setTimeoutInMilliseconds(int timeoutInMilliseconds) {
```

Comment:
good with the setter being non-nullable?

Reply:
yup